### DISCUSSION TITLE
Changed e-mail but receive e-mails on old address

### DISCUSSION THREAD

**Discussion by RianneDeHeide** [community/community#36773] [Other Feature Feedback, Questions, & Ideas]
Dear all,

I've changed universities and changed my e-mail address in the GitHub settings to my new one. Still, I get messages (that a collaborator pushed something in one of my repositories) on my old e-mail address (which I completely deleted in the settings). How to fix this?

Kind regards,
Rianne

**Comment by ossanna16** [comment]
Thank you for asking a good question @RianneDeHeide, I'm glad you found our community 🙂
